src/main/java/edu/greenblitz/robotName/subsystems/arm/elbow/FalconElbow/FalconElbowConstants.java

```diff
@@ -1,14 +1,28 @@
 package edu.greenblitz.robotName.subsystems.arm.elbow.FalconElbow;
 
 import com.ctre.phoenix6.configs.*;
+import com.ctre.phoenix6.signals.ForwardLimitTypeValue;
 import com.ctre.phoenix6.signals.NeutralModeValue;
+import com.ctre.phoenix6.signals.ReverseLimitTypeValue;
 import edu.wpi.first.math.controller.SimpleMotorFeedforward;
 
 import static edu.greenblitz.robotName.subsystems.arm.elbow.ElbowConstants.*;
 
 public class FalconElbowConstants {
 
-    public static final int MOTOR_ID = 1;
+    public static final int FALCON_MOTOR_ID = 1;
```

Comment:
why Falcon motor id? its the constants of the falcon sub subsystem. so it needs to be just motor id

Reply:
Was resolved without change?